packages/template-retail-react-app/package.json

```diff
@@ -93,5 +92,8 @@
       "path": "build/vendor.js",
       "maxSize": "320 kB"
     }
-  ]
+  ],
+  "overrides": {
+    "nwsapi": "2.2.2"
```

Comment:
we need to enforce this package in Retail App for the tests in generated project to pass. This should only affect unit tests, and not impact product code

Reply:
i wish json support comments